### ISSUE TITLE
Moves By Maxon Lower Jaw Distorting when mouth is open more than a small amount

### ISSUE THREAD

**Issue by iamYork667** [daz3d/DazToC4D#45]
When using Moves by Maxon on Daz model lower jaw distorts to the left heavy when mouth data opens more than a simple mumble...
![Capture](https://user-images.githubusercontent.com/98002440/202565480-f5764151-0cac-4f8c-bd3e-770e0634c9df.PNG)

**Comment by choerte** [comment]
It's not limited to Moves, but also affects any morph that is connected to the lower jaw. 
I found out that some of the eye morphs are also messed up, the up/down and side/side just makes them move diagonally. 

![moprhs-daz](https://user-images.githubusercontent.com/75386129/202896453-2ffc31b1-528f-490e-b560-2e2eb668ed83.jpg)

**Comment by santimagames** [comment]
Yes, I do have the exact same issue. I'm using the lastest version (2022 rev 12.25) of the plugin, Daz 4.21 and C4D 2023.1.3.  I also tried exporting in Fbx 2014 (default) and Fbx 2020 which made no difference. This applies to both 8.1 males and females. I thought the problem was just in the jaw area; come to think of it, the eyes look weird too.

**Comment by santimagames** [comment]
One more thing: Blink eye (left & right) is also exported badly:
![Screenshot_11](https://user-images.githubusercontent.com/35533390/211001897-8dd7e4b1-1b29-4af1-84aa-22e49aa3dac8.png)
